index.html

```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card">  
+            <div class="">
+              <img src="static/images/Snapshoot Portfolio 2.png" alt="">
+            </div>
+            <div>
+              <h2 class="works-title">Multi-Post Stories</h2>
+            </div>
+            <div class="card_details">
+              <ul>
+                  <li class="canopy">CANOPY</li>
+                   <li>Back End Dev</li>
+                   <li>2015</li>
+              </ul>
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card">  
+            <div class="">
+              <img src="static/images/Snapshoot Portfolio 3.png" alt="">
+            </div>
+            <div>
+              <h2 class="works-title">Tonic</h2>
+            </div>
+            <div class="card_details">
+              <ul>
+                  <li class="canopy">CANOPY</li>
+                   <li>Back End Dev</li>
+                   <li>2015</li>
+              </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card">
+            <div>
+              <img src="static/images/Snapshoot Portfolio 4.png" alt="">
+            </div>
+            <div>
+              <h2 class="works-title">Multi-Post Stories</h2>
+              <div class="card_details">
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                     <li>Back End Dev</li>
+                     <li>2015</li>
+                </ul>
+               
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
```

Comment:
- [ ] This must be a `section` tag as mentioned in the requirements. kindly consider applying this change.
- The container of the section is a `section` tag